runtime/gc_trace/TgcHeap.cpp

```diff
@@ -286,7 +286,7 @@ tgcShowRegions(OMR_VMThread *omrVMThread, const char *comment)
 	tgcExtensions->printf(">>> free region count %d\n", countFree);
 
 	countTotal += countMultiFree = regionPool->getMultiFreeList()->getTotalRegions();
-	tgcExtensions->printf(">>> mutliFree region count %d\n", countMultiFree);
+	tgcExtensions->printf(">>> multiFree region count %d\n", countMultiFree);
```

Comment:
This is code. Not sure of possible side affects, let's leave this for later.

Reply:
Oops, I did try to exclude such things, but I'm not perfect. Will drop.